demisto_sdk/commands/validate/file_validator.py

```diff
@@ -271,14 +273,40 @@ def get_modified_and_added_files(self, tag='origin/master'):
             print_ignored_files=self.print_ignored_files)
 
         if not self.is_circle:
-            files_string = run_command('git diff --name-status --no-merges HEAD')
-            nc_modified_files, nc_added_files, nc_deleted_files, nc_old_format_files = self.get_modified_files(
-                files_string, print_ignored_files=self.print_ignored_files)
-
-            all_changed_files_string = run_command('git diff --name-status {}'.format(tag))
-            modified_files_from_tag, added_files_from_tag, _, _ = \
-                self.get_modified_files(all_changed_files_string,
-                                        print_ignored_files=self.print_ignored_files)
+            remote_configured = has_remote_configured()
+            is_origin_demisto = is_origin_content_repo()
+            if remote_configured and not is_origin_demisto:
+                files_string = run_command('git diff --name-status --no-merges upstream/master...HEAD')
+                nc_modified_files, nc_added_files, nc_deleted_files, nc_old_format_files = \
+                    self.get_modified_files(files_string, print_ignored_files=self.print_ignored_files)
+
+                all_changed_files_string = run_command(
+                    'git diff --name-status upstream/master...HEAD')
+                modified_files_from_tag, added_files_from_tag, _, _ = \
+                    self.get_modified_files(all_changed_files_string,
+                                            print_ignored_files=self.print_ignored_files)
+            else:
+                remote_configured = False
+                is_origin_demisto = False
```

Comment:
what do you set them to false here and right after you do an if on the values you just set.
this looks like a bug.